client/layout/masterbar/checkout.tsx

```diff
@@ -9,12 +9,14 @@ import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopp
 import useValidCheckoutBackUrl from 'calypso/my-sites/checkout/composite-checkout/hooks/use-valid-checkout-back-url';
 import { leaveCheckout } from 'calypso/my-sites/checkout/composite-checkout/lib/leave-checkout';
 import useCartKey from 'calypso/my-sites/checkout/use-cart-key';
+import { QueryArgs } from 'calypso/my-sites/plans/jetpack-plans/types';
```

Comment:
Same as below, this import can be stripped by the compiler:

```suggestion
import type { QueryArgs } from 'calypso/my-sites/plans/jetpack-plans/types';
```